src/core/include/openvino/core/function.hpp

```diff
@@ -182,10 +184,12 @@ class OPENVINO_API Function : public std::enable_shared_from_this<Function> {
     virtual bool visit_attributes(ov::AttributeVisitor& visitor);
 
     /// Return the function parameters
+    OPENVINO_DEPRECATED("This member was deprecated. Please use inputs() instead.")
     const ov::ParameterVector& get_parameters() const {
         return m_parameters;
     };
     /// Return a list of function's outputs
+    OPENVINO_DEPRECATED("This member was deprecated. Please use outputs() instead.")
```

Comment:
A bit confusing without renaming Result op to Output

Reply:
ouputs() doesn't return operations, so we don't need to rename Parameter/Result operations

Reply:
is it worth to be done in the Model? So it would have inputs and outputs, but the function would have results and parameters? or this was discussed without me any you ruled that out?

Reply:
it's just to understand the impact. Added labels don't review, don't merge